commit 11d5bb9072ed7f73dd4b35017f11472e51cff8d1
Author: Alex Eagle <alex@aspect.dev>
Date:   Wed Jul 5 12:31:08 2023 -0700

    Revert "Make googleapis work on FreeBSD."
    
    This reverts commit 62b1d302e2fb4af5646ead1eb135aaf5b3c1a0e4.

diff --git a/go/private/repositories.bzl b/go/private/repositories.bzl
index 44c2da30..467c9054 100644
--- a/go/private/repositories.bzl
+++ b/go/private/repositories.bzl
@@ -262,7 +262,7 @@ def go_rules_dependencies(force = False):
             # releaser:patch-cmd gazelle -repo_root .
             Label("//third_party:go_googleapis-gazelle.patch"),
         ],
-        patch_args = ["-E", "-p1"],
+        patch_args = ["-p1"],
     )
 
     # releaser:upgrade-dep golang mock
